lib/puppet/provider/docker_compose/ruby.rb

```diff
@@ -9,7 +9,7 @@
 
   has_command(:docker, 'docker')
 
-  has_command(:dockercompose, 'docker-compose')
+  has_command(:dockercompose, 'docker')
```

Comment:
Since all usage of `dockercompose` is now replaced by `docker`, this line can be removed.